modular_skyrat/modules/emotes/code/scream_datums.dm

```diff
@@ -65,6 +65,11 @@ GLOBAL_LIST_EMPTY(scream_types)
 	male_screamsounds = list('sound/voice/ethereal/ethereal_scream_1.ogg', 'sound/voice/ethereal/ethereal_scream_2.ogg', 'sound/voice/ethereal/ethereal_scream_3.ogg')
 	female_screamsounds = null
 
+/datum/scream_type/plasmaman //Added tgstation#73828 screams to our modular scream solution
+	name = "Plasmaman Scream"
+	male_screamsounds = list('sound/voice/plasmaman/plasmeme_scream_1.ogg', 'sound/voice/plasmaman/plasmeme_scream_2.ogg', 'sound/voice/plasmaman/plasmeme_scream_3.ogg')
+	female_screamsounds = null
```

Comment:
I don't think that it's necessarily a good thing that this becomes available to everyone. I'm still thinking of auditing the scream types so that certain screams (such as this one) are either species-locked, or plainly no longer available.

Reply:
Without a refactor of the scream emote, if we remove the plasmaman scream from this datum, it makes it unselectable. Only non-player plasmamen will use the scream.

Reply:
I can make this donator only, if you'd like.